pkg/operator/exec.go

```diff
@@ -0,0 +1,26 @@
+package operator
+
+import (
+	"github.com/eclipse/che-operator/pkg/util"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	pgCommand = "psql -c \"CREATE USER keycloak WITH PASSWORD '" + keycloakPostgresPassword + "'\" " +
+		"&& psql -c \"CREATE DATABASE keycloak\" " +
+		"&& psql -c \"GRANT ALL PRIVILEGES ON DATABASE keycloak TO keycloak\" " +
+		"&& psql -c \"ALTER USER " + chePostgresUser + " WITH SUPERUSER\""
+)
+
+func ExecIntoPod(podName string, provisionCommand string) {
+	k8s := GetK8SConfig()
+	command := []string{"/bin/bash", "-c", provisionCommand}
+	logrus.Infof("Provisioning resources in pod %s", podName)
```

Comment:
Copied from somewhere else? Why?

Reply:
Because it says `provisioning resources in pod` but function name is `ExecIntoPod` and exec doesn't mean provisioning of any resource. The function name and log messages need to be aligned to avoid confusion.